readability-research.md

```diff
@@ -0,0 +1,127 @@
+# Review state of the art research papers about readability formulas
+
+
+## Common readability scales
+- Flesch Reading Ease
+- Flesch–Kincaid Grade Level
+- Automated Readability Index
+- New Dale–Chall Readability Formula
+- Advantage-TASA Open Standard (commercial)
+- Lexile (commercial)
+- Pearson Reading Maturity Metric (commercial)
+- TextEvaluator Tool (commercial)
+
+
+## Paper 1: A large-scaled corpus for assessing text readability
```

Comment:
My takeway from this is:

- Recent research leverages NLP to assess readability, which shows promising quality, but wouldn’t be suitable for us to include as a built-in feature in Wagtail. It could be done as a package though.
- Automated Readability Index is the worst of the models that’s been evaluated, but not by that much.

![figure 2 correlation plots](https://github.com/user-attachments/assets/5ea4602d-1a6f-4589-b67e-267c3f47dee5)